docs/powergate/devnet.md

```diff
@@ -0,0 +1,169 @@
+
+### Create a deal and store a file
+
+Now that your Powergate is running on devnet, all the CLI and API commands are the same as using it in production mode, just your deals will store faster (and disappear when you delete the devnet).
+
+**Install the CLI**
+
+From the root of the `powergate` repo, you can build the CLI from the latest code. This will install the Powergate CLI, `pow`, on your machine.
+
+```bash
+make build
+```
+
+Test your installation.
+
+```bash
+pow --help
+```
+
+**Create an FFS instance**
+
+FFS is the most common API for interacting with the Powergate. To use the API, you must first create an empty FFS instance, which will,
+
+- Create a new wallet address on Lotus.
+- Track and manage deals associated with that address in the Powergate FFS.
+- Create an API token for using that FFS (and address) over the Powergate API.
+
+```Bash
+pow ffs create
+```
+
+???+ success
+
+    ```Bash
+    Instance created with id 0ac0fb4d-581c-4276-bd90-a9aa30dd4cb4 and token 883f57b1-4e66-47f8-b291-7cf8b10f6370
+    ```
+
+**Generate a CID for your file on IPFS**
+
+To store data on the cold layer (Filecoin), you first need to make it available on IPFS. You can do that quickly by using the Powergate to temporarily add it to the embedded IPFS node.
+
+```bash
+pow ffs addToHot -t 883f57b1-4e66-47f8-b291-7cf8b10f6370 myfile
+```
+
+???+ success
+
+    ```Bash
+    Success! Added file to FFS hot storage with cid: QmYaAK8SSsKJsJdtahCbUe7MZzQdkPBybFCcQJJ3dKZpfm
+    ```
+
+Note: `addToHot` is not a permanent function, without further action it will only `add` the file to IPFS temporarily.
+
+**Push a storage config**
+
+Every storage deal in your FFS is managed by a storage `config`. To move the file from temporary storage to managed storage on the hot (IPFS) and cold (Filecoin) layers, we can push a new config for our existing CID generated above.
+
+Every FFS has a default `config` that can be used for every new deal, for now, we'll just use the default.
+
+```bash
+❯ pow ffs push -w -t 883f57b1-4e66-47f8-b291-7cf8b10f6370 QmYaAK8SSsKJsJdtahCbUe7MZzQdkPBybFCcQJJ3dKZpfm
+```
+
+???+ success
+
+    ```Bash
+    Success! Pushed cid config for QmYaAK8SSsKJsJdtahCbUe7MZzQdkPBybFCcQJJ3dKZpfm to FFS with job id: 966dcb44-9ef4-4d2a-9c90-a8103c77c354
+               JOB ID                   STATUS
+    966dcb44-9ef4-4d2a-9c90-a8103c77c354    Success
+    ```
+
+**Retrieve file from network**
+
+Finally, you can verify that the file was stored on the devnet by making a request to get it back out. 
+
+```bash
+❯ pow ffs get  -t 883f57b1-4e66-47f8-b291-7cf8b10f6370 QmYaAK8SSsKJsJdtahCbUe7MZzQdkPBybFCcQJJ3dKZpfm myfile2
+```
+
+???+ success
+
+    ```Bash
+    Success! Data written to myfile2
+    ```
+
+## Devnet with stand-alone Lotus node
+
+You can run the devnet to make use of a stand-alone [Lotus](https://lotu.sh/) with all the benefits described above. 
```

Comment:
Are you refering here to use the Lotus CLI?
If that's the case, maybe change to: `You can run the devnet to make use of the [Lotus] CLI`.
I can put some extra example here that may be cool too. Or maybe those asciicinema videos.

Reply:
done